AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -159,33 +172,33 @@ You can record to Alembic ABC files using the Alembic Recorder Clip. The followi
 * Particle (ParticleSystem)
 * Cameras (Camera)
 
-To configure the Timeline to record an object to Alembic you need to define the scope of the recording. By default the entire scene will be recorded but you can scope it to a object by setting the object as a Scope Target by setting the properties on Alembic Recorder Clip component:
+To configure the Timeline to record an object to Alembic you need to define the scope of the recording. By default the entire scene will be recorded but you can scope it to a hierarchy by setting the root object as a Scope Target in the properties on the Alembic Recorder Clip component:
 
 ![Alembic Shot Clip](images/abc_recorder_clip.png)
 
 | Property:| Function: |
 |:---|:---| 
-|__Output Path__ |{TODO}|
-|__Archive Type__ |{TODO} |
-|__Xform Type__ |{TODO} |
-|__Swap Handedness__ |{TODO} |
-|__Swap Faces__ |{TODO} |
-|__Swap Factor__ |{TODO} |
-|__Capture Scope__ |{TODO} |
-|__Assume None Skinned Meshes Are Constant__ |{TODO} |
-|__Capture MeshRenderer__ |{TODO} |
-|__Capture SkinnedMeshRenderer__ |{TODO} |
-|__Capture Particle__ |{TODO} |
-|__Capture Camera__ |{TODO} |
-|__Mesh Components Normals__ |{TODO} |
-|__Mesh Components UV1__ |{TODO} |
-|__Mesh Components UV2__ |{TODO} |
-|__Mesh Components Vertex Color__ |{TODO} |
-|__Mesh Components Submeshes__ |{TODO} |
-|__Ignore First Frame__ |{TODO} |
-|__Detailed Log__ |{TODO} |
-
-The effect the recording press "Play". The ABC file will be recorded to the output path which by default is outside the Project Asset folder. You can then bring the Alembic file back into the Project and play it back using the Timeline.
+|__Output Path__ |Specify the location where the Alembic Exporter will save the ABC file. By default the output path is relative to the current Unity project path. |
```

Comment:
The text here should match the text for the exporter. If the dialogs are identical we should just have a link here (see "exporter" dialog for more details on what the options do..)

Reply:
there are some options that are on the exporter component that aren't available here, should we still link to the component?

Reply:
No link for now then, let Amy figure it out. Just make sure the text is the same for common options for now

Reply:
yes I copy pasted it